CONTRIBUTING.rst

```diff
@@ -146,6 +146,57 @@ The resulting ISO will be stored in ``./result/iso`` directory.
 
 Note: You can put additional RPMs to ``./result/build/01-rpm-build`` and these will be automatically used for the ISO build.
 
+Local development workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This workflow makes it possible to test changes to the Anaconda source code locally on your machine without any dependencies
+on external infrastructure. It uses two scripts, one called ``scripts/rebuild_boot_iso`` to build a fresh bootable installation image (boot.iso)
+from Anaconda source code on the given branch and corresponding Fedora/CentOS Stream packages. The second script, called ``scripts/update_boot_iso``
+uses the Anaconda updates image mechanism together with the ``mkksiso`` command provided by the Lorax project to very quickly
+create an updated version of the boot.iso when Anaconda code is changed. The updated boot.iso can then be booted on a VM or bare metal.
+
+The ``rebuild_boot_iso`` script
```

Comment:
I would prefer that the script explanation is inside the script on the top.

Reply:
I've added a small note to each script noting what it does & where to find more. I kinda like to have both scripts documented in a single place to which I can direct people to.